migrator/readme.md

```diff
@@ -0,0 +1,49 @@
+## Running a migration
+
+The migrator sub project produces a executable jar which will migrate data from an R2 database into the path manager.
+
+To run you will need a `migrator.properties` file in the same directory you are executing from. The format of this file is:
```

Comment:
A few other projects place this file in S3 and then have a `fetch-config.sh` script to bring it down. Is it worth doing similar here?

Reply:
I just lifted the migrator stuff out of the main readme as I didn't think it was something we were likely to use any time soon but didn't want to delete it.